app/components/stepper-signup.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action, set } from '@ember/object';
+
+export default class StepperSignupComponent extends Component {
+  constructor() {
+    super(...arguments);
+
+    this.currentStep = Number(
+      +new URLSearchParams(window.location.search).get('step')
```

Comment:
We are already doing + again why do we need Number

Reply:
Yes Doing because making new flow of onboarding 
So We decided work separately from existing flow